documentation fails to build

**Describe the bug**
After recent changes in how we use `fmt`, documentation fails to build:
```
-- Building without Kokkos kernels library
CMake Warning at lib/checkpoint/src/CMakeLists.txt:147 (message):
  Checkpoint: Install targets skipped because CMake version 3.10.2<3.13;
  affects packages built with a Checkpoint install.  CMake <3.13 does not
  support exporting targets from subdirectories


-- Checkpoint: building as a sub-directory
(...)
-- Performing Test LINKER_SUPPORTS_RDYNAMIC - Success
CMake Warning at src/CMakeLists.txt:308 (message):
  VT: Install targets skipped because CMake version 3.10.2<3.13; affects
  packages built with a VT install.  CMake <3.13 does not support exporting
  targets from subdirectories


-- VT: building examples
(...)
-- Configuring done
CMake Error: install(EXPORT "vt" ...) includes target "vt" which requires target "fmt" that is not in the export set.
-- Generating done
```

Note: documentation build is the only one that uses CMake 3.10.

**To Reproduce**
```
docker-compose run ubuntu-docs
```

**Expected behavior**
Fix the build (either upgrade CMake version or work around this in CMakeLists.txt).

--- a/ci/build_cpp.sh
+++ b/ci/build_cpp.sh
@@ -46,17 +46,24 @@ if test -d "${source_dir}/lib/detector"
 then
     { echo "Detector already in lib... not downloading, building, and installing"; } 2>/dev/null
 else
-    git clone -b "${detector_rev}" --depth 1 https://github.com/DARMA-tasking/detector.git
-    export DETECTOR=$PWD/detector
-    export DETECTOR_BUILD=${build_dir}/detector
-    mkdir -p "$DETECTOR_BUILD"
-    cd "$DETECTOR_BUILD"
-    mkdir build
-    cd build
-    cmake -G "${CMAKE_GENERATOR:-Ninja}" \
-          -DCMAKE_INSTALL_PREFIX="$DETECTOR_BUILD/install" \
-          "$DETECTOR"
-    cmake --build . --target install
+    if test "${VT_DOXYGEN_ENABLED:-0}" -eq 1
+    then
+        cd "${source_dir}/lib"
+        git clone -b "${detector_rev}" --depth 1 https://github.com/DARMA-tasking/detector.git
+        cd -
+    else
+      git clone -b "${detector_rev}" --depth 1 https://github.com/DARMA-tasking/detector.git
+      export DETECTOR=$PWD/detector
+      export DETECTOR_BUILD=${build_dir}/detector
+      mkdir -p "$DETECTOR_BUILD"
+      cd "$DETECTOR_BUILD"
+      mkdir build
+      cd build
+      cmake -G "${CMAKE_GENERATOR:-Ninja}" \
+            -DCMAKE_INSTALL_PREFIX="$DETECTOR_BUILD/install" \
+            "$DETECTOR"
+      cmake --build . --target install
+    fi
 fi
 
 if test -d "${source_dir}/lib/checkpoint"
@@ -67,7 +74,7 @@ else
     then
         cd "${source_dir}/lib"
         git clone -b "${checkpoint_rev}" --depth 1 https://github.com/DARMA-tasking/checkpoint.git
-        cd ../../
+        cd -
     else
         git clone -b "${checkpoint_rev}" --depth 1 https://github.com/DARMA-tasking/checkpoint.git
         export CHECKPOINT=$PWD/checkpoint

--- a/ci/docker/ubuntu-18.04-gnu-docs.dockerfile
+++ b/ci/docker/ubuntu-18.04-gnu-docs.dockerfile
@@ -14,7 +14,6 @@ RUN apt-get update -y -q && \
     apt-get install -y -q --no-install-recommends \
     ca-certificates \
     curl \
-    cmake \
     git \
     mpich \
     libmpich-dev \
@@ -38,6 +37,11 @@ ENV MPI_EXTRA_FLAGS="" \
     CC=mpicc \
     CXX=mpicxx \
     PATH=/usr/lib/ccache/:$PATH
+
+COPY ./ci/deps/cmake.sh cmake.sh
+RUN ./cmake.sh 3.18.4
+
+ENV PATH=/cmake/bin/:$PATH
 
 FROM base as build
 COPY . /vt

